zerver/webhooks/stripe/view.py

```diff
@@ -53,6 +55,14 @@ def api_stripe_webhook(
     payload: Dict[str, Any] = REQ(argument_type="body"),
     stream: str = REQ(default="test"),
 ) -> HttpResponse:
+    sig_header = request.META["HTTP_STRIPE_SIGNATURE"]
+    try:
+        stripe.Webhook.construct_event(request.body, sig_header, get_secret("stripe_api_endpoint"))
```

Comment:
I think it should be `stripe_webhook_endpoint_secret`, not `stripe_api_endpoint`.